src/Catty/DataModel/Project/Project.m

```diff
@@ -319,17 +161,17 @@ - (BOOL)isEqualToProject:(Project*)project
         return NO;
     if (! [self.userData isEqual:project.userData])
         return NO;
-    if ([self.objectList count] != [project.objectList count])
+    if ([self.scene.objects count] != [project.scene.objects count])
```

Comment:
The actual content of the scene should be compared in the `isEqual` method of the `Scene` itself. Hence here it is enough to simply call this method of the scene.